test/Mono.Linker.Tests.Cases/Inheritance.Interfaces/OnReferenceType/NoKeptCtorButInterfaceNeeded/NestedInterfaces1.cs

```diff
@@ -0,0 +1,52 @@
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+
+namespace Mono.Linker.Tests.Cases.Inheritance.Interfaces.OnReferenceType.NoKeptCtorButInterfaceNeeded {
+	public class NestedInterfaces1 {
```

Comment:
@marek-safar fyi - of all the tests I added in this PR this is the only one that failed.

Reply:
`Foo` was having it's `Method()` stripped resulting in invalid IL